contracts/lib/Licensing.sol

```diff
@@ -28,10 +29,19 @@ library Licensing {
     /// license is burnt for linking
     /// @param licensorIpId Id of the IP this license is for
     /// @param transferable Whether or not the license is transferable
+    // TODO: the struct will not be used in mainnet.  will remove.
     struct License {
         uint256 policyId;
         address licensorIpId;
         bool transferable;
         // TODO: support for transfer hooks
     }
+
+    struct MintingLicenseConfig {
```

Comment:
how to hook up external mint engine like manifold?

Reply:
Mainfold should integrate with our LicenseNFT contract. will consider this feature with separated PR.